src/acuvity/apexextend.py

```diff
@@ -106,116 +157,42 @@ async def scan_async(
-        Returns a Policeresponse object on success, and raises different exceptions on failure.
-
-        This function does **NOT** allow to use different analyzers or redactions as policies are being **managed** by the Acuvity backend.
-        To configure different analyzers and redactions you must do so in the Acuvity backend.
-
-        :param messages: the messages to scan. These are the prompts that you want to scan. Required if no files or a direct request object are provided.
-        :param files: the files to scan. These are the files that you want to scan. Required if no messages are provided. Can be used in addition to messages.        :param request_type: the type of the validation. This can be either Type.INPUT or Type.OUTPUT. Defaults to Type.INPUT. Use Type.OUTPUT if you want to run model output through the detection engines.
-        :param annotations: the annotations to use. These are the annotations that you want to use. If not provided, no annotations will be used.
-        :param bypass_hash: the bypass hash to use. This is the hash that you want to use to bypass the detection engines. If not provided, no bypass hash will be used.
-        :param anonymization: the anonymization to use. This is the anonymization that you want to use. If not provided, but the returned detections contain redactions, then the system will use the internal defaults for anonymization which is subject to change.
-        :param provider: the provider to use. This is the provider name that you want to use for policy resolutions. If not provided, it will default to the principal name (the application itself).
-        :param user: the user to use. This is the user name and their claims that you want to use.
-        """
-        return await self.police_request_async(request=self.__build_police_request(
-            *messages,
-            files=files,
-            request_type=request_type,
-            annotations=annotations,
-            bypass_hash=bypass_hash,
-            anonymization=anonymization,
-            provider=provider,
-            user=user,
-        ))
+        guard_config: GuardConfig,
+    ) -> Scanrequest:
+        request = Scanrequest.model_construct()
 
-    def __build_police_request(
-        self, # pylint: disable=unused-argument
-        *messages: str,
-        files: Union[Sequence[Union[str,os.PathLike]], os.PathLike, str, None] = None,
-        request_type: Union[Type,str] = Type.INPUT,
-        annotations: Optional[Dict[str, str]] = None,
-        bypass_hash: Optional[str] = None,
-        anonymization: Union[Anonymization, str, None] = None,
-        provider: Optional[str] = None,
-        user: Optional[Union[Policeexternaluser,Tuple[str, List[str]],Dict[str, Any]]] = None,
-    ) -> Policerequest:
-        request = Policerequest.model_construct()
+        keywords = keywords or []
+        redactions = redactions or []
```

Comment:
please remove this here and use them again as proper optional types. deal with none/not none in the guard_config section.